superset/assets/javascripts/explorev2/main.css

```diff
@@ -34,3 +34,39 @@
 .background-transparent {
   background-color: transparent !important;
 }
+.viztype-selector-container {
```

Comment:
we could use react-bootstrap's grid system for layout and probably get rid of some of this css
https://react-bootstrap.github.io/components.html#grid